etc/config/c++.amazon.properties

```diff
@@ -3,7 +3,7 @@ compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&clang-rocm:&mosclang-trunk:&rv
 # The disabled groups are actually used in the c++.gpu.properties. One day these might exist on both servers, so I want
 # to keep them in the same place.
 defaultCompiler=g132
-demangler=/opt/compiler-explorer/gcc-13.1.0/bin/c++filt
+demangler=/opt/compiler-explorer/clang-trunk/bin/llvm-cxxfilt
```

Comment:
Quick question, do we want to change the default demangler for all C++ compilers to llvm-cxxfilt? I'm a tiny bit worried this might break something since it's been gcc's c++filt for so long

Reply:
it should definitely be only for clang compilers, this line should stay the same

Reply:
Fixed. Thanks!